website/docs/docs/on-premises-2x/environment-variables.md

```diff
@@ -643,4 +650,35 @@ Disables the new user interface introduced in version 2.0.0. When this environme
 
 **Required:** false<br/>
 **Default:** unset<br/>
-**Allowed values:** `true`<br/>
\ No newline at end of file
+**Allowed values:** `true`<br/>
```

Comment:
I believe this can also be added to 1.* onprem env variables, unless you want to only advertise this as a 2.x feature ;)